docs/5-user-authentication.md

```diff
@@ -1,163 +1,42 @@
 # 5. User authentication
 
-> AngularFire2 allows you configure your authentication methods in the bootstrap
-phase of your application. This means that if your app only uses one form
-of authentication, you can specify that ahead of time so you only need to call
-`af.auth.login()` later, and with no parameters.
+> AngularFire2 provides you an Observable for your Firebase Authentication
+State via the `AngularFireAuth` module.
 
-## Configure application in bootstrap
-
-To specify your authentication ahead of time, you provide the `AngularFireModule.initializeApp` function
-with an `AuthProvider` and an `AuthMethod`.
-
-```ts
-import { BrowserModule } from '@angular/platform-browser';
-import { NgModule } from '@angular/core';
-import { AppComponent } from './app.component';
-import { AngularFireModule, AuthProviders, AuthMethods } from 'angularfire2';
-
-const myFirebaseConfig = {
-  apiKey: '<your-key>',
-  authDomain: '<your-project-authdomain>',
-  databaseURL: '<your-database-URL>',
-  storageBucket: '<your-storage-bucket>',
-  messagingSenderId: '<your-messaging-sender-id>'
-};
-
-const myFirebaseAuthConfig = {
-  provider: AuthProviders.Google,
-  method: AuthMethods.Redirect
-};
-
-@NgModule({
-  imports: [
-    BrowserModule,
-    AngularFireModule.initializeApp(myFirebaseConfig, myFirebaseAuthConfig)
-  ],
-  declarations: [ AppComponent ],
-  bootstrap: [ AppComponent ]
-})
-export class AppModule {}
-```
-
-
-## Login users
-
-If you have setup authentication in bootstrap like above, then all you need to do
-is call login on `af.auth.login()`
-
-The lone exception is if you're using username and password, then you'll have
-to call `af.auth.login()` with the user's credentials.
-
-```ts
-af.auth.login({ email: 'email', password: 'pass' });
-```
+AngularFireAuth's `.auth` object will return an initialized
+`firebase.auth.Auth` instance, allowing you to log users in and out. [See
+the Firebase docs for more information on what methods are availabile.](https://firebase.google.com/docs/reference/js/firebase.auth.Auth)
 
 **Example app:**
 
 ```ts
 import { Component } from '@angular/core';
-import { AngularFire } from 'angularfire2';
+import { Observable } from 'rxjs/Observable';
+import { AngularFireAuth } from 'angularfire2/auth';
+import { GoogleAuthProvider, User as FirebaseUser } from 'firebase/auth';
```

Comment:
When I try to import `User as FirebaseUser` from `firebase/auth` the CLI tells me: `Cannot find name 'FirebaseUser'.`

I changed the type of `user` to: `Observable<firebase.User>` and this worked.